src/containers/Careers.js

```diff
@@ -15,25 +19,122 @@ const CareersContainer = styled.div`
   flex-direction: column;
 `;
 
+const SwingWrapper = styled.div`
+  position: relative;
+  height: 70%;
+  width: 80%;
+  margin: 0 2rem;
+  z-index: 0;
+  overflow: visible;
+`;
+
+let swipeCount = 1;
+
 class Careers extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      stack: null,
+    };
+  }
+
   componentDidMount() {
     this.props.fetchCareers();
   }
 
+  swingShow = () => {
+    if (
+      this.props.careers.dataFetched &&
+      this.props.careers.careerCards.length
+    ) {
+      return (
+        <Swing
+          tagName="div"
+          setStack={stack => this.setState({ stack: stack })}
+          ref="stack"
+          throwout={e => {
+            swipeCount++; // someone please suggest a better way to do this
+            this.likeCareer(e);
+          }}
+          config={{
+            allowedDirections: [Direction.LEFT, Direction.RIGHT],
+            maxRotation: 5,
+            minThrowOutDistance: 1000,
+            maxThrowOutDistance: 1400,
+            throwOutConfidence: (xOffset, yOffset, element) => {
+              const xConfidence = Math.min(
```

Comment:
Why have you decided to call it `xConfidence` and `yConfidence`? I'm not sure if I understand the naming... :flushed:

Reply:
This was taken from the swing module docs so it was easier to throw the card and the user didn't need to drag it really close to the edge of the screen